usagov/usagov-2021
# USAGOV-1164: TXNDATA failure during bootstrap.sh

https://cm-jira.usa.gov/browse/USAGOV-1164

## Description
Added in verbose information on what `bootstrap.sh` is doing, so when this TXNDATA error happens again, we can confirm which line was the cause with absolute certainty.
A notice about the potential TXNDATA error has also been noted in the script's logging.

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions
We will need to deploy again, and review the output from `bootstrap.sh` - we will then take further action from there. You can confirm with Dale Frey while testing this.

## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
